packages/models/src/transactions/transaction-error.model.ts

```diff
@@ -1,7 +1,12 @@
+export enum TransactionErrors {
```

Comment:
Introducing an `enum` for these as otherwise I need to add `// eslint-disable-next-line lingui/no-unlocalized-strings` wherever I am throwing the errors in the mobile app